docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -23,7 +23,29 @@ If the user name is `Alice` and the password is `secret`, the HTTP authorization
 ====
  
 The Basic Authentication mechanism does not provide confidentiality protection for the transmitted credentials. The credentials are merely encoded with Base64 when in transit and not encrypted or hashed in any way. Therefore, Basic Authentication is used with HTTPS to provide confidentiality.
- 
+
+There are a few limitations associated with Basic Authentication, which include:
+
+Credentials as plain text::
++
+--
+It is required to use HTTPS with Basic Authentication. However, if the load balancer terminates HTTPS, the risk of exposing credentials as plain text increases when HTTP is forwarded to Quarkus.
+
+Also, in multi-hop deployments, HTTPS is only used between client and the first Quarkus endpoint, and the basic credentials are propagated from the first Quarkus endpoint to the next endpoint. Therefore, the credentials are sent as plain text.
```

Comment:
```suggestion
Also, in multi-hop deployments, the credentials can be exposed if HTTPS is used between the client and the first Quarkus endpoint only but they are propagated to the next endpoint over HTTP.
```

Reply:
Done, changed the sentence as follows:

"Also, in multi-hop deployments, the credentials can be exposed if HTTPS is used between the client and the first Quarkus endpoint only. However, the credentials are propagated to the next Quarkus endpoint over HTTP."